target/linux/ramips/dts/KE2P.dts

```diff
@@ -0,0 +1,127 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "phicomm,ke2p", "mediatek,mt7621-soc";
+	model = "Phicomm KE 2P";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_blue;
+		led-running = &led_blue;
+		led-upgrade = &led_blue;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		stat_r {
+			label = "ke2p:red:status";
+			gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+		};
+
+		stat_y {
+			label = "ke2p:yellow:status";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+
+		led_blue: stat_b {
+			label = "ke2p:blue:status";
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
```

Comment:
Please change to "gpio-keys" and remove poll-interval.

Reply:
True.